notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>An interactive version of this would be great, if time permits _ i'd like to be able to zoom into the data to see how well the simulations agree over various timescales</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I think we can plan for adding interactive plots in a future version of the notebook. </p>